src/org/ods/orchestration/phases/DeployOdsComponent.groovy

```diff
@@ -32,55 +31,52 @@ class DeployOdsComponent {
     }
 
     @TypeChecked(TypeCheckingMode.SKIP)
-    public void run(Map repo, String baseDir) {
+    void run(Map repo) {
```

Comment:
NOTE: This method is basically unchanged. The huge diff is only due to the removal of `steps.dir(baseDir) {}`. Switching to the correct working dir is done by the surrounding code now to reduce code duplication.